quota-management/README.md

```diff
@@ -0,0 +1,214 @@
+# Sample Quota Management Strategy for OpenShift
+
+The following is an example of a strategy that might be employed by an Operations team for managing project quotas. This approach uses the concept of _tier profiles_ so as to avoid having to manage many custom quotas for each project/tenant. In this example we have defined the following tier profiles.
+
+| Profile | Project CPU             | Project Memory          | Per Pod CPU                       | Per Pod Memory                         |
+|---------|-------------------------|-------------------------|-----------------------------------|----------------------------------------|
+| Small (default)  | 1 core (burstable to 2) | 6 Gi (burstable to 8)   | 10m - 100m; default request: 50m  | 128 Mi - 1 Gi; default request: 256 Mi |
+| Medium  | 2 core (burstable to 4) | 12 Gi (burstable to 16) | 10m - 200m; default request: 50m  | 128 Mi - 2 Gi; default request: 256 Mi |
+| Large   | 4 core (burstable to 8) | 24 Gi (burstable to 32) | 10m - 1000m; default request: 50m | 128 Mi - 4 Gi; default request: 256 Mi |
+
+The specs above are not important, nor are the number of profiles we've defined. What is important is how we define & apply the quotas to projects. We describe the process below.
+
+## Concepts and Workflow
+
+The following sections describe how we piece together this approach.
+
+### Quota Files
+
+First off, each quota is definied in its own YAML file (`./files/quota-small.yml`, `./files/quota-medium.yml`, `./files/quota-large.yml`) in a git repo. Each file contains the same _List_ of objects all with the same names:
```

Comment:
@etsauer I haven't had a chance to fully review, and I am planning to do so, but just noticed a quick typo above - i.e.: `definied`